pep-0688.rst

```diff
@@ -133,13 +134,16 @@ code, the interpreter extracts the underlying ``Py_buffer`` from the
 implements ``bf_getbuffer``, the returned buffer is wrapped in a
 ``memoryview`` for consumption by Python code.
 
-The ``__release_buffer__`` method corresponds to the
-``bf_releasebuffer`` C slot, which is called when a caller no
-longer needs the buffer returned by ``bf_getbuffer``. This is an
+The ``__release_buffer__`` method is called when a caller no
+longer needs the buffer returned by ``__buffer__``. It corresponds to the
+``bf_releasebuffer`` C slot. This is an
 optional part of the buffer protocol, and read-only buffers can
 generally omit it. The Python signature for this method is
 ``def __release_buffer__(self, buffer: memoryview, /) -> None: ...``.
-The buffer to be released is wrapped in a ``memoryview``. It is
+The buffer to be released is wrapped in a ``memoryview``. When this
+method is invoked through the buffer API (for example, through
+``memoryview.release``), the passed ``memoryview`` is the same object
```

Comment:
I think this isn't true, that it is the same `memoryview` when returned by a C client (e.g. `memoryview.release`), but that there is an obligation on a Python client using the Python API to unsure that it returns the same `memoryview`it was given. Greater length in discussion at https://discuss.python.org/t/pep-688-take-2-making-the-buffer-protocol-accessible-in-python/19756/4?u=jeff5 .

Reply:
This is resolved for me in the conversation leading up to https://discuss.python.org/t/pep-688-take-2-making-the-buffer-protocol-accessible-in-python/19756/9?u=jeff5 where the penny drops. Thanks!